### PR TITLE
add Nordic BLE sniffer pcap support

### PR DESCRIPTION
**Pull request by pccr10001**
I bought a [Nordic BLE sniffer](https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF-Sniffer).
Then I edited some files to support Nordic's pcap format.

### CODE REVIEW

File: crackle.c [hunk 1]
```diff
@@ -1286,6 +1303,9 @@ int main(int argc, char **argv) {
         case PPI:
                 packet_handler = packet_handler_ppi;
                 break;
+	case NORDIC_BLE_SNIFFER_META:
+		packet_handler = packet_handler_nordic;
+		break;
```
Comment (avsej) [inline crackle.c]: it does not seem to have consistent indentation (new code uses tabs)

Comment (pccr10001) [inline crackle.c]: I am sorry about that.
I will fix it soon.

Comment (rickypan108) [reply crackle.c]: Hi, has this been fixed and does crackle now support nordic BLE captured packets? The crackel version i have gives an error and isn't able to read packet capture (which has the pairing information) from Adafruit BLE sniffer ( based on nordic). Thanks , Rick

Comment (pccr10001) [reply crackle.c]: Hi, I will create a new pull request for the new version of crackle.

**Pull request closed without merging**